modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -107,9 +107,7 @@ class SeqexecEngine(settings: SeqexecEngine.Settings) {
         obsId <- safeGetObsId(id)
         _ <- systems.odb.obsAbort(obsId, e.msg)
       } yield ()
-      case (EventSystem(Executed(id)), st) if st.sequences.get(id).map(
-        _.status === SequenceState.Idle
-      ).getOrElse(false) => for {
+      case (EventSystem(Executed(id)), st) if st.sequences.get(id).exists(_.status === SequenceState.Idle) => for {
```

Comment:
I know this is carried over from the previous code, but `EventSystem(Executed(id))` can be written as `Event.executed(id)`.  Now that I think about it, maybe we should make `EventSystem` private to the engine package, it only makes sense internally.